### PR TITLE
UI Design and firebase setup

### PR DESCRIPTION
**Pull request by VijayaP-Cuelogic**
Please review the updates

### CODE REVIEW

File: app/src/main/java/com/example/cuelogicinventoryassignment/ActivityDeviceDetails.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.example.cuelogicinventoryassignment
+
+import android.os.Bundle
+import android.support.wearable.activity.WearableActivity
+
+class ActivityDeviceDetails : WearableActivity() {
```
Comment (anandl-cuelogic) [inline app/src/main/java/com/example/cuelogicinventoryassignment/ActivityDeviceDetails.kt]: Why you added wearableActivity

File: app/src/main/java/com/example/cuelogicinventoryassignment/AddNewDeviceActivity.kt [hunk 2]
```diff
@@ -0,0 +1,15 @@
+package com.example.cuelogicinventoryassignment
+
+import android.os.Bundle
+import android.support.wearable.activity.WearableActivity
+
+class AddNewDeviceActivity : WearableActivity() {
```
Comment (anandl-cuelogic) [inline app/src/main/java/com/example/cuelogicinventoryassignment/AddNewDeviceActivity.kt]: Please remove WearableActivity, we are not going to create Android wearable app

File: app/src/main/res/layout/activity_add_new_device.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.wear.widget.BoxInsetLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (anandl-cuelogic) [inline app/src/main/res/layout/activity_add_new_device.xml]: this layout use for wearable UI creation

**Review by anandl-cuelogic — CHANGES_REQUESTED** [review]

**Comment by VijayaP-Cuelogic** [comment]
committed required changes

File: app/src/main/res/layout/activity_add_new_device.xml (hunk 3 above)
Comment (VijayaP-Cuelogic) [reply app/src/main/res/layout/activity_add_new_device.xml]: removed boxInsetLayout

File: app/src/main/java/com/example/cuelogicinventoryassignment/ActivityDeviceDetails.kt (hunk 1 above)
Comment (VijayaP-Cuelogic) [reply app/src/main/java/com/example/cuelogicinventoryassignment/ActivityDeviceDetails.kt]: changed to AppCompatActivity

**Pull request merged**